apps/charterafrica/src/lib/data/common/processPageConsultation.js

```diff
@@ -68,13 +68,13 @@ async function processPageConsultationDocument(page, api, context) {
 
 async function processPageConsultation(page, api, context) {
   const { params } = context;
-
-  // Check if we are on a document page: /opportunities/consultation/documents
-  if (params.slugs.length > 2 && params.slugs[2] === "documents") {
+  // Check if we are on a document page: /opportunities/consultation/document
+  if (params.slugs.length > 2 && params.slugs[2] === "document") {
```

Comment:
@kelvinkipruto USe optional chaining for this just in case slugs can be undefined

Reply:
`slugs` have already been checked in `common/index.js` so at this point, they can not be `undefined`.